packages/artillery-plugin-memory-inspector/index.js

```diff
@@ -0,0 +1,63 @@
+const pidusage = require('pidusage');
+const debug = require('debug')('plugin:memory-inspector');
+
+function ArtilleryPluginMemoryInspector(script, events) {
+  this.script = script;
+  this.events = events;
+
+  const inspectorConfig =
+    script.config['memory-inspector'] ||
+    script.config.plugins['memory-inspector'];
+
+  async function memoryInspectorHandler(context, ee, next) {
+    if (typeof process.env.ARTILLERY_INTROSPECT_MEMORY !== 'undefined') {
```

Comment:
Likely a nitpick, but favor early returns. 

- Less scope nesting;
- Easier to read;
- Expresses the intention clearly.

Reply:
Hey @kettanaito ! 💯 agree- I also favor early returns as a general rule. I didn't here because I wouldn't be returning from this part of the logic, as the part below still has to run. So the option would be to move this logic below it, and then write

```js
if (typeof process.env.ARTILLERY_INTROSPECT_MEMORY === 'undefined') {
return done()
}
```
followed by then writing the logic below it.

So in this case, I didn't feel like I gained any readable gains or expressed the intent more clearly, especially since this wasn't particularly nested. But if you have a specific suggestion in mind that you feel would make it more readable, I'm happy to review this. Otherwise, I'd leave this one as is!

Reply:
My bad, overlooked the part below.